packages/panzer/dof-mgr/src/Panzer_DOFManager_impl.hpp

```diff
@@ -1304,7 +1322,10 @@ fillGIDsFromOverlappedMV(const ElementBlockAccess & access,
   using Teuchos::ArrayRCP;
 
   //To generate elementGIDs we need to go through all of the local elements.
-  ArrayRCP<ArrayRCP<const GO> > twoview = overlap_mv.get2dView();
+  auto dual_view = overlap_mv.getDualView();
```

Comment:
See above.  Get Kokkos::View objects from the Tpetra::MultiVector directly; don't access the DualView.